syntax = "proto3";

package pb;

option go_package = "./pb";


message MessageRequest{
    string name = 1;
}

message MessageReply{
    string message = 1;
}

message Message2Request{
    string name = 1;
}

message Message2Reply{
    string message = 1;
}

message Message3Request{
    string name = 1;
}

message Message3Reply{
    string message = 1;

}

service Greeter{
    rpc SayHello(MessageRequest) returns (MessageReply){}
    rpc SayHello2(Message2Request) returns (Message2Reply){}
    rpc SayHello3(Message3Request) returns (Message3Reply){}
}